Source/Core/DolphinQt/Config/Mapping/IOWindow.cpp

```diff
@@ -145,17 +162,38 @@ void IOWindow::ConnectWidgets()
           this, &IOWindow::OnRangeChanged);
   connect(m_range_slider, static_cast<void (QSlider::*)(int value)>(&QSlider::valueChanged), this,
           &IOWindow::OnRangeChanged);
+
+  connect(m_expression_text, &QPlainTextEdit::textChanged, [this] {
+    m_apply_button->setText(m_apply_button->text().remove(QStringLiteral("*")));
+    m_apply_button->setText(m_apply_button->text() + QStringLiteral("*"));
```

Comment:
This looks...weird. Why does it have to be done this way (not to mention: why at all?)
Is this trying to add an asterisk as "hey, something changed!" indicator at the end of the apply text?

Reply:
Yes. That is what it's doing. :)

Reply:
Kinda gets the job done, but feels a little roundabout...but I cannot really think of a better way to do this though other than moving it to a method (or providing a shown/hidden text instead of changing the button text)